api/src/pcapi/routes/backoffice/autocomplete.py

```diff
@@ -391,6 +391,51 @@ def autocomplete_criteria() -> AutocompleteResponse:
     )
 
 
+def _get_offerer_tag_base_query() -> sa_orm.Query:
+    return db.session.query(offerers_models.OffererTag).options(
+        sa_orm.load_only(
+            offerers_models.OffererTag.id,
+            offerers_models.OffererTag.label,
+            offerers_models.OffererTag.name,  # a tag should always have a label but we take the name as a fallback
+        )
+    )
+
+
+def prefill_offerer_tag_choices(autocomplete_field: fields.PCTomSelectField) -> None:
+    if autocomplete_field.data:
+        tags = (
+            _get_offerer_tag_base_query()
+            .filter(offerers_models.OffererTag.id.in_(autocomplete_field.data))
+            .order_by(offerers_models.OffererTag.label)
+            .all()
+        )
+        autocomplete_field.choices = [(tag.id, tag.label or tag.name) for tag in tags]
+
+
+@blueprint.backoffice_web.route("/autocomplete/offerer_tags", methods=["GET"])
```

Comment:
Détail : dans les routes du BO, on utilise plutôt des `-`.
```suggestion
@blueprint.backoffice_web.route("/autocomplete/offerer-tags", methods=["GET"])
```